tensornetwork/contractors/custom_path_solvers/__init__.py

```diff
@@ -0,0 +1,6 @@
+from tensornetwork.contractors.custom_path_solvers import pathsolvers
+from tensornetwork.contractors.custom_path_solvers import nconinterface
+#pylint: disable=line-too-long
+from tensornetwork.contractors.custom_path_solvers.pathsolvers import greedy_cost_solve, greedy_size_solve, full_solve_complete, _full_solve_single, _reduce_nets
```

Comment:
Don't include private methods in the init import. Presumably, a user should never use them